### PR TITLE
Add more taskbar icon context menu entries

### PR DESCRIPTION
**Pull request by Noxis-source**
https://github.com/jliddev/WowUp/issues/67

### CODE REVIEW

File: WowUp.WPF/ViewModels/MainWindowViewModel.cs [hunk 1]
```diff
@@ -416,5 +421,62 @@ private bool HasWarcraftLocation()
...
             return wowLocations.Any(loc => !string.IsNullOrEmpty(loc));
         }
+        void ShowTab(string header)
+        {
+            if (Application.Current?.MainWindow == null)
+            {
+                return;
+            }
+
+            TaskbarIconClickCommand.Execute(null);
```
Comment (jliddev) [inline WowUp.WPF/ViewModels/MainWindowViewModel.cs:431]: This can just be OnTaskbarIconClick

File: WowUp.WPF/ViewModels/MainWindowViewModel.cs [hunk 2]
```diff
@@ -416,5 +421,62 @@ private bool HasWarcraftLocation()
...
+            SelectedTabIndex = TabItems.IndexOf(GetTabItemByHeader(header));
+        }
+        void ShowTab(TabItem item)
+        {
+            if (Application.Current?.MainWindow == null)
+            {
+                return;
+            }
+
+            TaskbarIconClickCommand.Execute(null);
```
Comment (jliddev) [inline WowUp.WPF/ViewModels/MainWindowViewModel.cs:441]: This can just be OnTaskbarIconClick

**Review by jliddev — COMMENTED** [review]
I made some very minor changes.
Can we add a divider above the quit button.
Cab we hide the "Update all addons" button when there arent any to update?
Can we disable the "An update is available" button after its clicked.
Maybe rename "An update is available" to "Update WowUp" or something more specific.

Overall looks good!

**Comment by Noxis-source** [comment]
I think I can do everything easily except for
> hide the "Update all addons" button

This would require a binding ideally. Also, should I be able to bind to AddonViewViewModel, I can get rid of nesting and directly bind to commands as well as implement skipped features. A simple way to do that is to use a property and just set it at one point. However, I do not really want to do that. Maybe it makes sense to introduce parent-child relations between view models. I will try to do that and show you the result :)

**Comment by Noxis-source** [comment]
Meh, I did it in several ways. There are too much shitty code or structural changes in any case. Besides, I found a couple of bugs that I really do not have the mood to dig into. I think I'll just wait for the electron release and see what I can do there.

**Pull request closed without merging**